examples/stubbing__resources/cypress/integration/spec.js

```diff
@@ -0,0 +1,45 @@
+/// <reference types="cypress" />
+var MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
+
+describe('Stub loading of resources', () => {
+  it('requested image can be exchanged', () => {
+    cy.server();
+    cy.route({
+      method: 'GET',
+      url: '/apirequest',
+      response: JSON.stringify({
+        src: "https://cypress.io/cypress.png"
+      })
+    });
+
+    cy.visit("index.html").then(() => {
+      cy.get("#img-container").then($element => {
+        var el = $element[0];
+        observeDOM(el, mutations => {
+          // red dot
+          mutations[0].addedNodes[0].src = "data:image/png;base64," +
```

Comment:
can you load the mocked image from a fixture file?

Reply:
Yes of course. I have added it in the same way I am doing it in my project.